From 82ad8d6d61669ab0a81f76349d65052f029da7f4 Mon Sep 17 00:00:00 2001
From: Lawrence Mok <lawrence.mok@amlogic.com>
Date: Mon, 13 Oct 2014 18:00:45 -0700
Subject: [PATCH 4950/5965] fix pink/mosaic video seek/play problem

caused by bad change in "45732b43 change ppmgr canvas index base and max canvases"

Change-Id: I3b1c2a9dd31e3421649dafe94046cf5f49ebcf17
---
 include/linux/amlogic/amports/canvas.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/amlogic/amports/canvas.h b/include/linux/amlogic/amports/canvas.h
index 768a655502d2..08082df65f97 100755
--- a/include/linux/amlogic/amports/canvas.h
+++ b/include/linux/amlogic/amports/canvas.h
@@ -80,7 +80,7 @@ typedef struct {
 #endif
 
 /*here ppmgr share the same canvas with deinterlace and mipi driver for m6*/
-#define PPMGR_CANVAS_INDEX 0x66
+#define PPMGR_CANVAS_INDEX 0x70
 #define PPMGR_DOUBLE_CANVAS_INDEX 0x74  //for double canvas use
 #define PPMGR_DEINTERLACE_BUF_CANVAS 0x77   /*for progressive mjpeg use*/
 
-- 
2.19.0

